dftimewolf/lib/recipes/manager.py

```diff
@@ -35,7 +35,12 @@ def _ReadRecipeFromFileObject(
     description = json_dict['description']
     del json_dict['description']
 
-    args = json_dict['args']
+    args = []
+    for arg_list in json_dict['args']:
+      arg = RecipeArgs(switch=arg_list[0],
+                             help_text=arg_list[1],
```

Comment:
fix indentation for the next two lines

Reply:
or  you may be able to unpack from the list directly into the dataclass

Reply:
Done. Went with just fixing the indentation, as I know there's going to be an optional element to this dataclass in a later PR, I can look into your other suggestion.